task/06-conditions-n-loops-tasks.js

```diff
@@ -251,7 +295,22 @@ function reverseInteger(num) {
  *   4916123456789012 => false
  */
 function isCreditCardNumber(ccn) {
-    throw new Error('Not implemented');
+    let digit, digits, flag, sum, i, len;
+    flag = true;
+    sum = 0;
+    digits = (ccn + '').split('').reverse();
+    for (i = 0, len = digits.length; i < len; i++) {
+        digit = digits[i];
+        digit = parseInt(digit, 10);
+        if ((flag = !flag)) {
```

Comment:
if (value1 = value2) is not correct,
into `if` statement should only condition, cannot be a variable assignment operation